mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -2521,6 +2521,16 @@ func.func @transfer_write_1d_scalable_mask(%arg0: memref<1x?xf32>, %vec: vector<
 
 // -----
 
+// CHECK-LABEL: func @transfer_write_tensor
+//       CHECK:   vector.transfer_write
```

Comment:
There is no "tensor" operation checked in the output here?

Reply:
Actually, `tensor.dim` only exist in intermediate process.